zookeeper-lock/src/main/java/com/netflix/conductor/zookeeper/config/ZookeeperConfiguration.java

```diff
@@ -0,0 +1,22 @@
+package com.netflix.conductor.zookeeper.config;
+
+import com.netflix.conductor.core.config.Configuration;
+import org.apache.curator.framework.CuratorFrameworkFactory;
+
+public interface ZookeeperConfiguration extends Configuration {
+    String ZK_CONNECTION = "zk.connection";
+    String ZK_SESSIONTIMEOUT_MS = "zk.sessionTimeoutMs";
```

Comment:
`workflow.zookeeper.lock.session.timeout.ms`

Reply:
`sessionTimeoutMs` is a Zookeeper property we're passing through. Keeping it as `workflow.zookeeper.lock.sessionTimeoutMs`.